src/client/components/general/Campaign/index.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import './campaign.scss'
+import Article from '../Article';
+
+function Campaign( { theme, article, image} ) {
+
+  return (
+    <section className='campaign'>
+        <div className='image-container'>
+          <img src={image} alt='Hands holding a cosmetic'></img>
```

Comment:
Maybe change alt to be a prop that you pass in the component, instead of being hardcoded